lms/djangoapps/grades/tasks.py

```diff
@@ -23,50 +23,40 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible):
+def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, **kwargs):
```

Comment:
Is there any reason we couldn't place all of these arguments inside of kwargs? That would also allow for a future change that removes pre-existing args.